### ISSUE TITLE
Create Plugin for 3.0 - error on DeActivation

### ISSUE THREAD

**Issue by maiconpinto** [croogo/croogo#814]
Hi, I am creating a plugin  [Support](https://github.com/maiconpinto/croogo-support-plugin) for croogo v3.0 . But show error on DeActivation:

```
Error: Class 'SupportActivation' not found
File /home/maiconpinto/croogo/vendor/croogo/croogo/Core/src/Plugin.php
Line: 688
```
On activation same error, but after reload page, the plugin was activated. Now, when I try to deactivate, it show error and doesn't work. 

I saw the file Plugin.php:688, and find:

```php
if (file_exists($configFile) && include $configFile) {
    $this->_PluginActivation = new $className; //688

    return $this->_PluginActivation;
}
```

How did not understand, I would like your help. 

Obs: I created this plugin with: `bin/cake bake plugin Support` in my app folder. Was setup composer.json of my app with new autoload: `"Support\\": "./plugins/Support/src"`.

**Comment by rchavik** [comment]
This is most likely related to autoloading setup. I think we have not properly port the activation|deactivation class autoloading in 3.x.  I'll try to look into this later today

**Comment by rchavik** [comment]
@maiconpinto 

Unfortunately fixing this is not straightforward. And I have no ETA at the moment.

Sorry

**Comment by maiconpinto** [comment]
No problem @rchavik 

How I can to help? There are something that I can to do to help?

**Comment by rchavik** [comment]
Well, a few things needs to be done for this to be fixed:

- `<Plugin>Activation.php` is not feasible with cake 3.x, because a plugin can be <vendor>/<package>.  So we might simply use `PluginActivation.php` as the default file name
- Choose the proper namespace for the class, eg `Example\Config\PluginActivation`, or `Croogo\Translate\Config\PluginActivation`
- Adjust `Croogo\Core\Plugin` so that it loads the correct file and able to instantiate it
- Adjust the `Example` and `Croogo\Translate` activation class, and port its content to cake 3.x

**Comment by rchavik** [comment]
Fixed by  2cdd885
